app/services/diagnostics/move_inspector.rb

```diff
@@ -171,6 +171,51 @@ def generate
         else
           @output << "(no profile associated with move)\n"
         end
+
+        @output << <<~ENDPER
+
+          PERSON ESCORT RECORD
+          --------
+        ENDPER
+
+        # NB: it is better to identify PERs via profile (not via move directly), as there are some older records which are
+        # only associated with a profile but not with a move
+        if move.profile&.person_escort_record.present?
+          move.profile.person_escort_record.tap do |per|
+            @output << "id:\t#{per.id}\n"
+            @output << "framework ver:\t#{per.framework&.version}\n"
```

Comment:
✂️  ver/versions :)

Reply:
"version" was a bit too long vs "ver", and it made the tab indenting a little ugly:

```
PERSON ESCORT RECORD
--------
id:	14f3744d-edf3-4ab0-af34-107bbebc0ed0
framework version:	1.3.0
framework_id:	098cec4e-e0b5-47c2-988d-16efbf600a4e
profile_id:	2bbb14e5-0868-4d5a-8dad-7ba825aea468
move_id:	f24e4449-61b7-425c-84a2-60be57dca5a1
```

vs

```
PERSON ESCORT RECORD
--------
id:	14f3744d-edf3-4ab0-af34-107bbebc0ed0
framework ver:	1.3.0
framework_id:	098cec4e-e0b5-47c2-988d-16efbf600a4e
profile_id:	2bbb14e5-0868-4d5a-8dad-7ba825aea468
move_id:	f24e4449-61b7-425c-84a2-60be57dca5a1
```

Reply:
but I'm not too fussy either way :-)

Reply:
ah ok fair enough, merge away